azure-iot-device/tests/common/pipeline/test_pipeline_stages_base.py

```diff
@@ -1455,6 +1455,249 @@ def test_no_matching_request_id(self, mocker, stage, pending_op, event, fake_uui
         assert not pending_op.completed
 
 
+@pytest.mark.describe(
+    "CoordinateRequestAndResponseStage - .handle_pipeline_event() -- Called with ConnectedEvent"
+)
+class TestCoordinateRequestAndResponseStageHandlePipelineEventWithConnectedEvent(
+    CoordinateRequestAndResponseStageTestConfig, StageHandlePipelineEventTestBase
+):
+    @pytest.fixture
+    def event(self):
```

Comment:
>event [](start = 8, length = 5)

connected_event

Reply:
No. It has to be "event" I believe for the inherited tests to work.

---
In reply to: [400444487](https://github.com/Azure/azure-iot-sdk-python/pull/520#discussion_r400444487) [](ancestors = 400444487)

Reply:
Yes, Carter is correct.  I wanted it to be connected_event, but the inherited tests all broke.

---
In reply to: [400459206](https://github.com/Azure/azure-iot-sdk-python/pull/520#discussion_r400459206) [](ancestors = 400459206,400444487)